src/03-modality-agnostic-files.md

```diff
@@ -349,13 +350,16 @@ Additional fields can include external behavioral measures relevant to the
 scan.
 For example vigilance questionnaire score administered after a resting
 state scan.
+All such included additional fields SHOULD be documented in an accompanying
+`scans.json` file that describes these fields in detail
+(see [Tabular files](02-common-principles.md#tabular-files)).
 
-Example:
+Example `scans.tsv`:
 
 ```Text
-filename  acq_time
-func/sub-control01_task-nback_bold.nii.gz 1877-06-15T13:45:30
-func/sub-control01_task-motor_bold.nii.gz 1877-06-15T13:55:33
+filename    acq_time
```

Comment:
Some spurious spaces or tab was added

```suggestion
filename acq_time
```
did on the phone, so may be it isn't a tab there... Not sure it is a tab, but it should be so people don't cut paste wrong characters